packages/gatsby-plugin-mdx/src/plugin-options.ts

```diff
@@ -51,10 +51,11 @@ export const enhanceMdxOptions: EnhanceMdxOptions = async (
 ) => {
   const options = defaultOptions(pluginOptions)
 
-  // Set jsxRuntime based on Gatsby project config
+  // Set jsxRuntime & jsxImportSource based on Gatsby project config
   const { config } = helpers.store.getState()
-  const { jsxRuntime } = config
+  const { jsxRuntime, jsxImportSource } = config
   options.mdxOptions.jsxRuntime = jsxRuntime || `classic`
+  options.mdxOptions.jsxImportSource = jsxImportSource || `react`
```

Comment:
Shouldn't this be `@mdx-js/react`?

Reply:
No default is `react`: https://mdxjs.com/packages/mdx/#optionsjsximportsource

Don't mix this up with `providerImportSource`, which in fact is `@mdx-js/react`

Reply:
ohhh, I see. Then nvm